package.json

```
@@ -34,8 +34,8 @@
     "prebuild": "run-s clean test",
     "build": "run-s clean esbuild:prod",
     "start": "run-s clean esbuild:dev",
-    "start-assisted-ui": "env-cmd ./scripts/start-assisted-ui.sh",
-    "start-uhc-portal": "./scripts/start-uhc-portal.sh",
+    "start-assisted-ui": "yarn --cwd ../assisted-ui start",
+    "start-uhc-portal": "yarn --cwd ../uhc-portal dev-server --env noproxy api-env=staging",
```

```suggestion
    "start-uhc-portal": "yarn --cwd ../uhc-portal start",
```
Their start script is already doing this: `dev-server --env noproxy api-env=staging`